astropy/io/misc/hdf5.py

```diff
@@ -178,10 +178,8 @@ def read_table_hdf5(input, path=None):
     else:
         # Read the meta-data from the file
         table.meta.update(input.attrs)
-
     return table
 
-
```

Comment:
Please revert these linebreak changes, pep8 recommend to use 2 empty lines between functions.